kqueen/engines/eks.py

```diff
@@ -70,18 +70,18 @@ class EksEngine(BaseEngine):
             }
         },
         'cluster': {
-            'node_count': {
-            # TODO unsupported, need to define template k8s input
-                'type': 'integer',
-                'label': 'Node Count',
-                'default': 3,
-                'order': 3,
-                'validators': {
-                    'required': True,
-                    'min': 1,
-                    'number': True
-                }
-            },
+#            'node_count': {
+#              TODO  currently unsupported due EKS API restrictions
```

Comment:
due to?
https://en.oxforddictionaries.com/definition/due
https://en.oxforddictionaries.com/definition/due_to